config/database.yml

```diff
@@ -20,11 +20,27 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  host: localhost
+  # Try to set default connection properties based on DATABASE_URL.
+  # Otherwise, default to trusted localhost connections on the default port.
+  <% if ENV['DATABASE_URL'] %>
+  <% db_uri = URI.parse(ENV['DATABASE_URL']) %>
+  host: <%= db_uri.host || 'localhost' %>
+  port: <%= db_uri.port || '!!null' %>
+  username: <%= db_uri.user || '!!null' %>
+  password: <%= db_uri.password || '!!null' %>
+  <% end %>
 
 development:
   <<: *default
+  # Try to set the database to the value specified in DATABASE_URL if we're
+  # actually in the expected environment. Otherwise, default to
+  # web-monitoring-db_development
+  <% if ENV['RAILS_ENV'] == 'development' && ENV['DATABASE_URL'] %>
```

Comment:
Checking `ENV['RAILS_ENV']` is not a good way to check the environment for two reasons:

- There are several ways an environment can get set in Rails, including the `RAILS_ENV` OR `RACK_ENV` environment variables, so this doesn’t adequately check whether we are actually in the development environment.

- Various method calls and situations within Rails can change the environment and I’m not totally certain they reliably update `ENV`.

To check the environment in Rails, you should use `Rails.env`, like so:

```rb
if Rails.env.development?
  # do whatever
end
```